src/planscape/plan/tests.py

```diff
@@ -478,7 +478,7 @@ def setUp(self):
         self.base_condition = BaseCondition.objects.create(
             condition_name="condition1", condition_level=ConditionLevel.ELEMENT)
         self.condition1 = Condition.objects.create(
-            condition_dataset=self.base_condition)
+            condition_dataset=self.base_condition, condition_score_type=0, is_raw=False)
```

Comment:
nit: is there an enum value that may be used here instead of 0?

Reply:
Done